casanovo/config.yaml

```diff
@@ -1,30 +1,50 @@
 ###
 # Casanovo configuration.
-# Blank entries are interpreted as "None"
+# Blank entries are interpreted as "None".
+# Parameters that can be modified when running inference with Casanovo,
+# i.e. denovo and eval modes in CLI, are marked with "(I)". Other parameters
+# shouldn't be changed unless a new Casanovo model is being trained.
 ###
 
 # Random seed to ensure reproducible results.
 random_seed: 454
 
 # Spectrum processing options.
+# Max number of peaks in spectrum
```

Comment:
Clarify that peaks are sorted by intensity, and the least intense peaks beyond this number are discarded.